src/frontend/src/router/routes.ts

```diff
@@ -24,4 +25,13 @@ export const routes = [
       redirect: get(auth_state) === AuthStates.REGISTERED ? "/posts" : "/login",
     }
   },
+  {
+    name: "/edit-profile",
+    layout: BasicLayout,
```

Comment:
tomu basic layoutu by to chcelo dať nejaký margin na spodku, pretože potom máme všetko nalepené na spodku obrazovky, ak je to dosť veľké